coral/src/app/features/approvals/schemas/components/schema-request-table.tsx

```diff
@@ -68,10 +56,10 @@ const columns: Array<DataTableColumn<SchemaRequestTableData>> = [
       return (
         <GhostButton
           onClick={() => alert("Approve")}
-          title={`Approve schema request for ${row.topicname}`}
-        >
-          <Icon color="grey-70" icon={tickCircle} />
-        </GhostButton>
+          aria-label={`Approve schema request for ${row.topicname}`}
+          icon={tickCircle}
+          dense
```

Comment:
🦅  👀 ! (which is good) 
Let me check!

Reply:
🤔  that looks weird, doesn't it? (only view details is dense now)

![example](https://user-images.githubusercontent.com/943800/218720592-af18cb39-aaf7-4ba2-a89f-c7ae225ecdff.jpg)

Reply:
But it matches the sizes in Figma - well, the grey icons, the other is 15.9x15.9 instead of 16.67x16.67 like in Figma, but 16.47 is not doable in DS components.

Reply:
Updated!